README.md

```diff
@@ -2,20 +2,73 @@
 
 Extendable file viewer for web
 
-Supported file formats:
- - png, jpeg
+## Supported file formats:
+
+ - png, jpeg, gif, bmp, including 360-degree images
+ - pdf
  - csv
  - xslx
  - docx
+ - mp4
+
+
+## Usage
+
+There is one main React component, `FileViewer`, that takes the following props:
+
+`fileType` string: type of resource to be shown (one of the supported file
+formats, eg `'png'`). Passing in an unsupported file type will result in the
+display of an `unsupported filetype` error.
```

Comment:
it doesn't display an error per say. Just rendered an unsupported file type component? I initially interpreted this as it throws an error